models/iterable__campaigns.sql

```diff
@@ -35,7 +35,7 @@ with campaign_event_metrics as (
 ), campaign_join as (
 
     {% set exclude_fields = [ 'campaign_id', 'template_id'] %} -- these are both in campaigns
-    {% set exclude_fields = exclude_fields | upper if target.type == 'snowflake' else exclude_fields %} -- snowflake needs uppercase :)
+    {% set exclude_fields = exclude_fields %} 
```

Comment:
We can ideally remove this line, correct? This `set` statement is doing the same as the above line.

Reply:
You're right, I didn't even realize it became redundant!